kubeflow-metadata-envoy-config.yaml

```diff
@@ -0,0 +1,24 @@
+package:
+  name: kubeflow-metadata-envoy-config
+  version: 2.0.4 # check if this should be the version
```

Comment:
what would 2.0.4 map to?  I'm wondering if we should start this at 1.0.0 so it has it's own version lifecycle?

Reply:
That's the version of the tag where the file came from in kubeflow/pipelines, but I think it'd be fair to version from 1.0.0.

Reply:
This is addressed by the latest changes.